OpenRA.Game/Traits/Player/FixedColorPalette.cs

```diff
@@ -17,7 +17,7 @@ namespace OpenRA.Traits
 	public class FixedColorPaletteInfo : ITraitInfo
 	{
 		[Desc("The name of the palette to base off.")]
```

Comment:
Can you please add "Defaults to the active TileSet's Palette definition if empty" to these descriptions? The descriptions still assume the fields all default to "terrain".